src/components/Sources.tsx

```diff
@@ -1,7 +1,22 @@
 import { Source, Layer } from "react-map-gl/maplibre";
-import PropTypes from "prop-types";
+import type {
+  FillLayerSpecification,
+  LineLayerSpecification,
+} from "maplibre-gl";
 
-export const ProvSource = ({ data, style }) => (
+type LayerPaintProperty = NonNullable<FillLayerSpecification["paint"]>;
+type FillOpacity = LayerPaintProperty["fill-opacity"];
+
+interface SourceWithFillLayerProps {
+  data: unknown;
```

Comment:
is unknown the best type here ?

Reply:
It's what the component demands, so I just imitated its API. I'd love to give it a more sophisticated type in the future just for the fun of it but I don't think that's what we need at the moment.

Reply:
note for the future: object is FORBIDDEN (dixit @juanigaray )